### ISSUE TITLE
About copy ssh channel

### ISSUE THREAD

**Issue by Kinmfer** [kingToolbox/WindTerm#2262]
I found that windterm has a problem with SSH channel replication, that is, all copied windows will share a channel, and if any one of the connections is disconnected, it will affect other connections in this channel. I have never encountered this in Tabby or Xshell. Can you please optimize the relevant logic? Thank you very much.

**Comment by kingToolbox** [comment]
Yes, for a long time the design of this feature has been that disconnecting a channel will disconnect the session that the channel belongs to, which will lead to all open channels in that session being closed simultaneously. Your issue reminded me that perhaps it would be enough to just close that channel. I will try a new implementation approach and release it in version `windterm 2.7.0 prerelease 2`, so please stay tuned. Thank you very much!

**Comment by Kinmfer** [comment]
I am really happy to have this feature. I like windterm very much and have recommended it to countless people. Thank you very much!!!
